netman/dbus.go

```diff
@@ -8,24 +8,71 @@ import (
 	"github.com/godbus/dbus"
 )
 
-func getDevices() []string {
+type Client struct {
```

Comment:
golang forces to have a comment for exported types like
// Client this struct holds a dbus object blah blah ...
Same for the rest of methods or types starting with uppercase